### DISCUSSION TITLE
Width/Height of screen

### DISCUSSION THREAD

**Discussion by LordVagous** [slint-ui/slint#4094] [Q&A]
How do I set the width and height of a window to half the screen?

**Comment by Horbin-Magician** [comment]
Maybe you need to call the system api to get the screen size or use other crates.

**Comment by ogoffart** [comment]
You need to use window().set_size  with the size which you can compute based on the screen size.  To get the screen size, you will need to use another crate.
For example you can get it with private API using the i-slint-backend-winit crate.
